content/press/press-content.md

```diff
@@ -1,11 +1,34 @@
 <!--This is the main content file to edit for this page. It is embedded in src/pages/press.js-->
 
-**Press:** [Contact](info@environmentalenforcementwatch.org) a member of EDGI's EEW project team.
 
-## Images from events
+## Resources for press/media personnel
```

Comment:
Do you think it would be better to swap the order of this page/put coverage first? I'm ok either way but am trying to imagine who might interact w this page and how, thinking reporters might be interested in both sections but other folks might only be interested in the press coverage section

Reply:
Yeah I think it might even be worth creating separate pages for press coverage and press resources